apps/web/src/Pages/Staking/Staking.tsx

```
@@ -100,43 +110,88 @@ function Staking(): ReactElement {
+                    title: "Understanding Voi Staking",
+                  },
+                  {
+                    slug: "airdrop-programs-everything-you-need-to-know-a84706bd8599",
+                    title: "Airdrop Programs: Everything You Need To Know",
+                  },
+                  {
+                    slug: "staking-program-how-to-guide-382ea5085dab",
+                    title: "Staking Program: How To Guide",
+                  },
+                ].map((item) => (
+                  <li style={{ padding: "5px", listStyle: "inside" }}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)